vizro-core/tests/tests_utils/asserts.py

```diff
@@ -27,7 +27,12 @@ def assert_component_equal(left, right, *, keys_to_strip=None):
     If keys_to_strip is set to STRIP_ALL then only the type and namespace of component
     will be compared, similar to doing isinstance.
 
-    Examples
+    Args:
+        left: Dash component to compare.
+        right: Dash component to compare.
+        keys_to_strip: Keys to strip from the component dictionary before comparison.
```

Comment:
If we add the args description, might be worth moving the doc description on keys_to_strip to the actual args description 👍 
```suggestion
        left: Built Dash component.
        right: Expected Dash component.
        keys_to_strip: Keys to strip from the component dictionary before comparison.  If keys_to_strip is set to STRIP_ALL then only the type and namespace of component will be compared, similar to doing isinstance.
```

Reply:
I like Li's suggestion. Just remove the double space character between "... comparison." and "If keys_to_strip ..." (which would probably be handled by linter)

Reply:
Great suggestion. I did not do the suggestion on left and right though, because I think that is too specific. While it is the general convention, the function also could handle the other way.